src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet"
+    href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+    crossorigin="anonymous" />
+</head>
+<body> 
+   <header>
+        <h2>ShowTime</h2>
+        <div id="counter">
+            <h2>Shows</h2>
+            <span id="shows-counter"></span>
+        </div>
+    </header>
+    <ul class="allShows"></ul>
+<div id="modal-popup" class="modal-view"></div>
```

Comment:
[OPTIONAL] It would be better to improve the UI design of the comment modal. I see an overlap between the comments and the movie's description.
<img width="945" alt="image" src="https://user-images.githubusercontent.com/77190203/158018239-1b45833e-4609-4ec2-a869-8700130457bf.png">

Reply:
> [OPTIONAL] It would be better to improve the UI design of the comment modal. I see an overlap between the comments and the movie's description. <img alt="image" width="945" src="https://user-images.githubusercontent.com/77190203/158018239-1b45833e-4609-4ec2-a869-8700130457bf.png">

Hi @medaminedev66 , Thank you for reviewing my project.
At 0:26 part of the video, I specifically said: " I connected the application to the API and that was part of my work and I am proud of it".